doc/devel/testing.rst

```diff
@@ -173,6 +173,17 @@ decorator:
      If some variation is expected in the image between runs, this
      value may be adjusted.
 
+Freetype version
+----------------
+
+Due to subtle differences in the font rendering under different
+version of freetype some care must be taken when generating the
+baseline images.  Currently, almost all of the images were generated
```

Comment:
"Currently (2015)"  

It might be a good idea to document when currently is? To make it more obvious if this is out of date